### ISSUE TITLE
Upgrading to org-ref3 is painful, and needs more explanation

### ISSUE THREAD

**Issue by Stwerp** [jkitchin/org-ref#926]
I don't think this is necessarily an issue with the package itself, but I'm completely lost in getting things working again (and I have deadlines).

I had a wonderful setup in the older org-ref version, and now I can't even get a citation. A more helpful walk through would benefit someone like me, but I'm not even sure where half my setup comes from at this point.

Issues I've had

- Previously, I could `C-]` to bring up the citation window. This was a keybind, and not a huge deal, ... but
- Previously, I could begin typing and then easily search through various databases (IEEE is the one I need!). This issue seemed to be gone. However, I was able to determine that binding `C-]` to `org-ref-cite-insert-helm` mostly brought back the functionality I wanted.
- Whereas, previously the bibtex cite keys were automatically done, whatever configuration I had is now broken in v3 and everything came in with the same cite key (the year). I think I can figure this out, but *sigh*.
- And now, I have been able to cite stuff, but my tex output is not even including the references section. Same org file and setup as before, but upgrading to `org-ref: Version 3.0` and `org-version: 9.4.6` is just completely broken.

Ok, maybe this is all fine and I'm not using the right org-version. I also remember there were MELPA/repository changes at some point.

Basically: I was on leave last year and had time to get everything setup wonderfully. I don't have time to go through it again now, and wow, its all breaking and confusing. Anything that can be done to ease the v2 to v3 changes of org-ref for dunce users like me?

**Comment by Stwerp** [comment]
Let me preface by: I am indebted to what you have done with your scimax items and org-ref. I owe you many beers, as org-ref and some of your early videos has caused me to entirely re-think how I do presentations and prep lecture material. So I should start by a huge Thank You for your work!.

I apologize for the rashness of this "issue" I opened. This semester has caused many things (network configs, remote work, normal workflows, my brain) to break, and its just frankly been hard to keep up and I was in a frantic point yesterday. Many apologies.

Yesterday I thought I had read through several of the links you had given, but was still confused. There are indeed a lot of resources available, and I just need to set aside some time to go through and really understand them. I admit I don't understand my own config (technically, whatever config I copied from scimax) well enough to break apart what was in `org-ref`, what was `org-helm` or `bilblio.el` or the other myriad of reference items.  To me, `org-ref` was an abstraction into references, and that abstraction interface changed. I now realize that much of what I was using came from `helm-bibtex`, and so copying over the helm section of your config bootstrapped me back to operational. Thank you for the help and your work.
